source/community/release_notes/ulmo/ulmo_frontend_plugin_slots.rst

```diff
@@ -0,0 +1,139 @@
+  This slot is used to replace/modify/hide the course video upload page UI.
+
+* Added `EditFileAlertsSlot <https://github.com/openedx/frontend-app-authoring/tree/release/ulmo/src/plugin-slots/EditFileAlertsSlot>`_:
+  This slot is used to add alerts to the course file edit page.
+
+* Added `EditVideoAlertsSlot <https://github.com/openedx/frontend-app-authoring/tree/release/ulmo/src/plugin-slots/EditVideoAlertsSlot>`_:
+  This slot is used to add alerts to the course video edit page.
+
+
+Learning MFE
+*********************************************
+
+* Added `ContentIFrameErrorSlot <https://github.com/openedx/frontend-app-learning/tree/release/ulmo/src/plugin-slots/ContentIFrameErrorSlot>`_:
+  This slot is used to replace/modify the content iframe error page.
+
+* Added `CourseExitViewCoursesPluginSlot <https://github.com/openedx/frontend-app-learning/tree/release/ulmo/src/plugin-slots/CourseExitPluginSlots/CourseExitViewCoursesPluginSlot>`_:
+  This slot is used for modifying "View Courses" button in the course exit screen
+
+* Added `DashboardFootnoteLinkPluginSlot <https://github.com/openedx/frontend-app-learning/tree/release/ulmo/src/plugin-slots/CourseExitPluginSlots/DashboardFootnoteLinkPluginSlot>`_:
+  This slot is used for modifying the link to the learner dashboard in the footnote on the course exit page
+
+* Added `SequenceNavigationSlot <https://github.com/openedx/frontend-app-learning/tree/release/ulmo/src/plugin-slots/SequenceNavigationSlot>`_:
+  This slot is used to replace/modify/hide the sequence navigation component that controls navigation between units within a course sequence.
+
+* Updated `CourseBreadcrumbsSlot <https://github.com/openedx/frontend-app-learning/tree/release/ulmo/src/plugin-slots/CourseBreadcrumbsSlot>`_:
+  Added ``pluginProps``.
+
+* Updated `NotificationTraySlot <https://github.com/openedx/frontend-app-learning/tree/release/ulmo/src/plugin-slots/NotificationTraySlot>`_:
+  Added ``pluginProps``.
+
+* Updated `UnitTitleSlot <https://github.com/openedx/frontend-app-learning/tree/release/ulmo/src/plugin-slots/UnitTitleSlot>`_:
+  Modified ``pluginProps`` behavior, ``isEnabledOutlineSidebar`` is now always `true`. 
+
+
+.. seealso::
+
+   * :doc:`/site_ops/how-tos/use-frontend-plugin-slots`
+
+   * :doc:`../sumac/customizing_learner_dashboard`
+
+   * :doc:`../sumac/customizing_header`
+
+**Maintenance chart**
+
++--------------+-------------------------------+----------------+--------------------------------+
+| Review Date  | Working Group Reviewer        |   Release      |Test situation                  |
++--------------+-------------------------------+----------------+--------------------------------+
+|              | Frontend WG                   | Ulmo           |                                |
++--------------+-------------------------------+----------------+--------------------------------+
```

Comment:
This is the main reason this PR is a draft.

Reply:
Gotcha. I would put today's date (it's accurate as of today) and either your name or "Frontend WG"